cloudbuild.yaml

```diff
@@ -374,6 +374,30 @@ steps:
     dir: cmd/sdk-server/bin
     args: [-m, cp, '*.zip', gs://agones-artifacts/sdk-server]
 
+  #
+  # Cleanup services "preview", "development", "default"
+  #
+  - name: gcr.io/cloud-builders/gcloud
+    id: cleanup-services
+    waitFor: [e2e-wait-to-become-leader]
+    allowFailure: true
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        set -e
+        set -o pipefail
+        for service in preview development default; do
+          echo "Cleaning up $service service..."
+          #Retrieve version ids
+          versions=$(gcloud app versions list --service=$service --filter="traffic_split<1" --sort-by=~last_deployed_time.datetime --format="table[no-heading](version.id)")
+           if [ "$service" == "preview" ]; then
+            tail -n +20 <<< "$versions"
```

Comment:
Let's just make it all 50 versions - on review there isn't an actual reason to have different amounts.

Reply:
I am removing the 'if' condition because I don't think it's necessary now. I've applied it previously in case we needed to remove a different number of versions, but if we're deleting 50 versions for all three services, this approach will also work.